frontend/components/Header/Links.js

```diff
@@ -4,14 +4,39 @@ import s from 'styled-components'
 import { Icon } from '../common'
 import { LOGIN_URL } from '../../utils'
 import { mediaMaxWidth, MD } from '../../constants/measurements'
-import { MEDIUM_GRAY, DARK_GRAY, BORDER } from '../../constants/colors'
+import {
+  SNOW,
+  SNOW_ALPHA,
+  CLUBS_RED,
+  MEDIUM_GRAY,
+  DARK_GRAY,
+  BORDER,
+} from '../../constants/colors'
 import { logEvent } from '../../utils/analytics'
 
 const StyledIcon = s(Icon)`
   opacity: 0.5;
   margin-right: 4px;
 `
 
+const LoginButton = s.a`
+  border: 0;
+  background-color: ${CLUBS_RED};
+  padding: 14px 20px;
+  margin: auto;
+  color: ${SNOW} !important;
+
+  &:hover {
+    color: ${SNOW_ALPHA} !important;
```

Comment:
Generally want to keep font color the same but then make the background color a slightly darker version. So feel free to make a color called CLUBS_RED_DARK or CLUBS_RED_DEEP. A good way to do this is to google the hex number then click a little bit to the bottom right of the color in the color picker until there is a noticeable but not extreme difference between the two colors. 

lmk if you need help with this.

Reply:
Done, though check out the hex number I chose and make sure I didn't go too far with the difference.